aqueduct/services/geostore_service.py

```diff
@@ -14,15 +14,16 @@ def execute(config):
             if not response or response.get('errors'):
                 raise GeostoreNotFound
             geostore = response.get('data', None).get('attributes', None)
-            geojson = geostore.get('geojson', None)
+            geojson = geostore.get('geojson', None).get('features', None)[0]
+            
         except Exception as e:
             raise GeostoreNotFound(message=str(e))
         return geojson
 
     @staticmethod
     def get(geostore):
         config = {
-            'uri': '/geostore/'+ geostore,
+            'uri': '/v1/geostore/'+ geostore,
```

Comment:
this feels like a breaking change to me. was it tested properly?

Reply:
Probably locally where versioning doesn't exist 🙄